corpus/convert.py

```diff
@@ -18,12 +18,12 @@
 Path = setup_path(__package__)
 
 
-class CorpusReplaceOrthFromReferenceCorpus(Job):
+class CorpusReplaceOrthFromReferenceCorpusJob(Job):
```

Comment:
Discussion question: the renaming will change the hash. I am not sure if only @Atticus1806 ever used this class, but if not, maybe we should keep the old name.

Reply:
nvm I had a different idea

Reply:
I would not introduce a second class. I would change the existing class name